metricbeat/module/ceph/df/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: df
+  type: group
+  description: >
+    df
+  fields:
+    - name: example
```

Comment:
Please, just to be sure: is there a way to generate fields automatically? Or just manually?